src/elements/tables/body.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+/**
+ * Empty component for the future purposes.
+ */
+const Body = ({ children, tag: Tag = 'tbody', ...restProps }) => (
```

Comment:
Could you please wrap it into React.useRef(). I want to observe this element. Actually would be nice to wrap all components.

Reply:
I suggest you to use pure HTML elements. Maybe it's better than complex API I created.

I should remove these components I guess. Although, [_HeaderCell_](https://github.com/oacore/design/pull/48#issuecomment-565966268) is good idea and we should have it.

Reply:
I like it and it's not that complex. It works just like a proxy. Let's keep it, please.

Reply:
Also, text inside `HeaderCell` should be bigger and bold than in `Cell`

Reply:
Why?

Reply:
Well because it's how you designed it. I mean the font style can be applied on TableHead instead.
![h](https://user-images.githubusercontent.com/12775440/70912182-b87eb300-2013-11ea-84e1-c19a410483e9.PNG)

Reply:
I might missed something. Thank you for pointing this up

Reply:
It should be 1rem (16px) everywhere for this moment. We will see how it works in the future.